examples/payment.rs

```diff
@@ -24,22 +27,27 @@ async fn main() {
 
     bot.start(|context, provider_token| async move {
         let call_result = if context.text.value == START_PARAMETER {
-            let price = [LabeledPrice::new(TITLE, 1_00)];
-            let mut invoice = context.send_invoice(
+            let price = [LabeledPrice::new(TITLE, 10_000)];
```

Comment:
```suggestion
            let price = [LabeledPrice::new(TITLE, 100_00)];
```

The underscore here was originally used to separate the fractional part from the integer one.